iwyu.cc

```diff
@@ -798,6 +799,11 @@ class BaseAstVisitor : public RecursiveASTVisitor<Derived> {
     if (FunctionDecl* fn_decl = DynCastFrom(expr->getDecl())) {
       if (!IsImplicitlyInstantiatedDfn(fn_decl))
         return true;
+      if (const auto* call_expr = current_ast_node_->GetAncestorAs<CallExpr>(
```

Comment:
The formatting on this block is pretty horrible. Seems hard to shorten it, but can you trick `clang-format` by making it longer? E.g. use `current_ast_node()` instead of the raw member?

You could also consider moving the comment line down into the body of the if, but I don't think that alone helps with the wrapping.